main.go

```diff
@@ -110,6 +112,24 @@ func mainWithExitCode() exitCode {
 		return code
 	}
 
+	if args.enableGoRuntimeProfiler {
+		addr, _ := strings.CutPrefix(args.agentURL, "http://")
+		err = profiler.Start(
+			profiler.WithService(args.serviceName+"-profiler"),
```

Comment:
```suggestion
			profiler.WithService("dd-otel-host-profiler"),
```

I think it would be fair to hardcode a single service name so we can easily aggregate the profiles, no strong opinion though

Reply:
Good idea, but `dd-otel-host-profiler` is already the default value for the service, maybe something like `dd-otel-host-profiler-profile` ?

Reply:
Good catch! That works for me, a couple shorter alternatives:
* dd-otel-host-self-profiler
* dd-otel-host-self-profile

No strong opinon, would be onboard with either

Reply:
I went for `dd-otel-host-self-profiler`